src/integrators/ResNN.jl

```diff
@@ -53,6 +53,12 @@ function  apply(this::ResNN{T},theta::Array{T},Y0::Array{T},doDerivative=true) w
     nex = div(length(Y0),nFeatIn(this))
     Y   = reshape(Y0,:,nex)
     tmp = Array{Any}(this.nt+1,2)
```

Comment:
@eldadHaber @eldadHaber 

Does `tmp` need to be an `Array{Any}`? It looks like `tmp` could be an `Array{Array{T}, 1}`